tool/src/main/java/org/datacommons/tool/Tool.java

```diff
@@ -0,0 +1,231 @@
+package org.datacommons.tool;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.datacommons.proto.Debug;
+import org.datacommons.proto.Mcf;
+import org.datacommons.util.McfParser;
+import org.datacommons.util.McfUtil;
+import org.datacommons.util.TmcfCsvParser;
+import picocli.CommandLine;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+// TODO: Add e2e tests once Debug.Log is fully plumbed in.
+@CommandLine.Command(
+    name = "dc-import",
+    mixinStandardHelpOptions = true,
+    version = "dc-import 0.1",
+    description = "Tool for use in developing datasets for Data Commons.",
+    subcommands = {Lint.class, GenMcf.class})
+class Tool {
+  @CommandLine.Option(
+      names = {"-o", "--outputDir"},
```

Comment:
nit: --output-dir? dash is used in most popular tools

Reply:
Done.